client/src/main/java/org/spine3/protobuf/Messages.java

```diff
@@ -121,14 +121,19 @@ public static Any toAny(TypeName type, ByteString value) {
      *
      * @param messageType full type name defined in the proto files
      * @return message class
-     * @throws ClassNotFoundException in case there is no corresponding class for the given Protobuf message type
+     * @throws RuntimeException wrapping {@link ClassNotFoundException} if there is no corresponding class
+     *                          for the given Protobuf message type
      * @see #fromAny(Any) that uses the same convention
      */
-    public static <T extends Message> Class<T> toMessageClass(TypeName messageType) throws ClassNotFoundException {
+    public static <T extends Message> Class<T> toMessageClass(TypeName messageType) {
         final ClassName className = TypeToClassMap.get(messageType);
-        @SuppressWarnings("unchecked")
-        final Class<T> result = (Class<T>) Class.forName(className.value());
-        return result;
+        try {
+            @SuppressWarnings("unchecked")
+            final Class<T> result = (Class<T>) Class.forName(className.value());
+            return result;
+        } catch (ClassNotFoundException e) {
+            throw propagate(e);
```

Comment:
In some places we simply propagate, but in other we throw `UnknownTypeException`. We need to be consistent in handling unknown types.